### DISCUSSION TITLE
Alternative command for argocd app terminate-op <APP> --core

### DISCUSSION THREAD

**Discussion by alita1991** [argoproj/argo-cd#20992] [Q&A]
Hi,

I'm looking for an alternative way to execute the following command using `kubectl` instead of `argocd`:

`argocd app terminate-op "<APP>" --core`

I found the official ArgoCD documentation for the sync operation, but couldn't find anything related to the terminate operation.

https://github.com/argoproj/argo-cd/blob/fc6105b2bd7de98cfdb8bbfc75f306ba5713cb03/docs/user-guide/sync-kubectl.md

Any assistance would be greatly appreciated.

Thanks

**Comment by pasha-codefresh** [comment]
You can see whole logic of termination here
https://github.com/argoproj/argo-cd/blob/030a7be7e2fd0ae976c9b73196d777ea72bada09/server/application/application.go#L2298

It is just `a.Status.OperationState.Phase = common.OperationTerminating` inside application resource

**Comment by martivo** [comment]
To anyone else not wating to go through golang code, the answer is
```
kubectl patch -n ${ARGOCD_NAMESPACE} app ${APP_NAME} --type merge --patch '{"status": {"operationState": {"phase": "Terminating"}}}'
```
